demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  name: caikit-tgis-runtime
+spec:
+  multiModel: false
+  supportedModelFormats:
+    # Note: this currently *only* supports caikit format models
+    - autoSelect: true
+      name: caikit
+  containers:
+    - name: kserve-container
+      image: quay.io/opendatahub/text-generation-inference:stable
+      command: ["text-generation-launcher"]
+      args: ["--model-name=/mnt/models/artifacts/"]
+      env:
+        - name: TRANSFORMERS_CACHE
+          value: /tmp/transformers_cache
+      # resources: # configure as required
+      #   requests:
+      #     cpu: 8
+      #     memory: 16Gi
+    - name: transformer-container
+      image: quay.io/opendatahub/caikit-tgis-serving:stable
+      env:
+        - name: RUNTIME_LOCAL_MODELS_DIR
+          value: /mnt/models
+      ports:
+        - containerPort: 8080
```

Comment:
@Xaenalt it doesn't work for me if I use port  8080, it returns me HTTP invalid request error in the pod logs. It works if I use port 8085. Could you pls double check?

Reply:
@bdattoma: try changing:

```diff
diff --git a/demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml b/demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml
index 8528909..9fee801 100644
--- a/demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml
+++ b/demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml
@@ -21,7 +21,7 @@ spec:
       #     cpu: 8
       #     memory: 16Gi
     - name: transformer-container
-      image: quay.io/opendatahub/caikit-tgis-serving:stable
+      image: quay.io/opendatahub/caikit-tgis-serving:fast
       env:
         - name: RUNTIME_LOCAL_MODELS_DIR
           value: /mnt/models
```

Reply:
See here:

https://github.com/caikit/caikit-nlp/pull/244/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R77-R82